### ISSUE TITLE
MissingPluginException with webview_flutter

### ISSUE THREAD

**Issue by Z6P0** [flutter/flutter#35964]
I'm using the "webview_flutter" plugin in my app. Most of the time it works, but sometimes I have this error:

```
[ERROR:flutter/lib/ui/ui_dart_state.cc(148)] Unhandled Exception: MissingPluginException(No implementation found for method loadUrl on channel plugins.flutter.io/webview_0)
#0      MethodChannel.invokeMethod (package:flutter/src/services/platform_channel.dart:314:7)
<asynchronous suspension>
#1      MethodChannelWebViewPlatform.loadUrl (package:webview_flutter/src/webview_method_channel.dart:52:21)
<asynchronous suspension>
#2      WebViewController.loadUrl (package:webview_flutter/webview_flutter.dart:446:39)
<asynchronous suspension>
#3      Engine.load (package:test_flutter/bug_webview.dart:152:24)
<asynchronous suspension>
#4      CustomList.build.<anonymous closure> (package:test_flutter/bug_webview.dart:94:16)
#5      GestureRecognizer.invokeCallback (package:flutter/src/gestures/recognizer.dart:182:24)
#6      TapGestureRecognizer._checkUp (package:flutter/src/gestures/tap.dart:365:11)
#7      TapGestureRecognizer.handlePrimaryPointer (package:flutter/src/gestures/tap.dart:275:7)
#8      PrimaryPointerGestureRecognizer.handleEvent (package:flutter/src/gestures/recognizer.dart:455:9)
#9      PointerRouter._dispatch (package:flutter/src/gestures/pointer_router.dart:75:13)
#10     PointerRouter.route (package:flutter/src/gestures/pointer_router.dart:102:11)
#11     _WidgetsFlutterBinding&BindingBase&GestureBinding.handleEvent (package:flutter/src/gestures/binding.dart:218:19)
#12     _WidgetsFlutterBinding&BindingBase&GestureBinding.dispatchEvent (package:flutter/src/gestures/binding.dart:198:22)
#13     _WidgetsFlutterBinding&BindingBase&GestureBinding._handlePointerEvent (package:flutter/src/gestures/binding.dart:156:7)
#14     _WidgetsFlutterBinding&BindingBase&GestureBinding._flushPointerEventQueue (package:flutter/src/gestures/binding.dart:102:7)
#15     _WidgetsFlutterBinding&BindingBase&GestureBinding._handlePointerDataPacket (package:flutter/src/gestures/binding.dart:86:7)
#16     _rootRunUnary (dart:async/zone.dart:1136:13)
#17     _CustomZone.runUnary (dart:async/zone.dart:1029:19)
#18     _CustomZone.runUnaryGuarded (dart:async/zone.dart:931:7)
#19     _invoke1 (dart:ui/hooks.dart:250:10)
#20     _dispatchPointerDataPacket (dart:ui/hooks.dart:159:5)
```

$ flutter doctor -v

```
[✓] Flutter (Channel stable, v1.7.8+hotfix.3, on Linux, locale fr_FR.UTF-8)
    • Flutter version 1.7.8+hotfix.3 at /opt/flutter
    • Framework revision b712a172f9 (il y a 2 jours), 2019-07-09 13:14:38 -0700
    • Engine revision 54ad777fd2
    • Dart version 2.4.0

 
[✓] Android toolchain - develop for Android devices (Android SDK version 28.0.3)
    • Android SDK at /home/*/Android/Sdk
    • Android NDK location not configured (optional; useful for native profiling
      support)
    • Platform android-Q, build-tools 28.0.3
    • ANDROID_HOME = /home/*/Android/Sdk
    • Java binary at: /opt/android-studio/jre/bin/java
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b16-5323222)
    • All Android licenses accepted.

[✓] Android Studio (version 3.4)
    • Android Studio at /opt/android-studio
    • Flutter plugin version 35.3.1
    • Dart plugin version 183.6270
    • Java version OpenJDK Runtime Environment (build
      1.8.0_152-release-1343-b16-5323222)

[✓] IntelliJ IDEA Ultimate Edition (version 2019.1)
    • IntelliJ at /opt/idea-IU-181.5281.24
    • Flutter plugin version 36.1.4
    • Dart plugin version 191.7830

[✓] Connected device (1 available)
    • Nokia 7 plus • * • android-arm64 • Android 8.1.0 (API 27)

• No issues found!
```

**Comment by kush3107** [comment]
Make sure to stop and rerun your app after you have run `flutter packages get`

**Comment by Z6P0** [comment]
@kush3107 Yes I did but it still crashes every time.

**Comment by jithin-pal** [comment]
@kush3107 the same error still exists even when hot restarted the application. Any fix?

**Comment by BondarenkoStas** [comment]
@Z6P0 Can you please try it with current flutter stable v1.9.1+hotfix.2 and latest webview_flutter?

**Comment by iapicca** [comment]
Without additional information, we are unfortunately not sure how to resolve this issue. We are therefore reluctantly going to close this bug for now. Please don't hesitate to comment on the bug if you have any more information for us; we will reopen it right away!
  
Thanks for your contribution.

**Comment by javad-vovin** [comment]
I think the issue is caused by webview being not visible, I wrap webview in a visibility widget and showing error state on onWebResourceError, webview throws exception when I call any of it's methods.

**Comment by iapicca** [comment]
@javad-vovin 
this issue was closed more than a year ago and it's not actively worked on
if you are experiencing this error please file a new issue
providing your `flutter doctor -v` your `flutter run -v` and a minimal reproducible code sample

**Comment by javad-vovin** [comment]
Yes I know it's closed, Someone else created new issue, #70655
I wanted to provide more information if someone else encountered with this.
Thanks
